datalad/support/tests/test_annexrepo.py

```diff
@@ -1029,8 +1031,8 @@ def test_annex_ssh(repo_path, remote_1_path, remote_2_path):
     rm2 = AnnexRepo(remote_2_path, create=False)
 
     # check whether we are the first to use these sockets:
-    socket_1 = opj(ssh_manager.socket_dir, 'datalad-test')
-    socket_2 = opj(ssh_manager.socket_dir, 'localhost')
+    socket_1 = opj(ssh_manager.socket_dir, get_connection_hash('datalad-test'))
+    socket_2 = opj(ssh_manager.socket_dir, get_connection_hash('localhost'))
```

Comment:
I feel like having `ssh_manager.get_connection_hash` doing this opj could be of help ;)